clang/test/CodeGen/profile-continuous.c

```diff
@@ -0,0 +1,16 @@
+// RUN: %clang %s -S -emit-llvm -fprofile-generate -fprofile-continuous -o - | FileCheck %s --check-prefix=IRPGO
```

Comment:
Without a fixed target triple I believe this command will fail on certain platforms that do not support PGO/coverage.

We typically add `%clang` tests to test/Driver and `%clang_cc1` tests to test/CodeGen, for better layering separation

Reply:
In this test, I'm testing both the driver (lines 4,5,8) and the clang codegen (lines 1,2,3,6).
Should I split it into 2 files, one for each stage?

Reply:
split out the Driver and Codegen tests. Used triples in the Driver but not in Codegen because I see existing IRPGO [tests](https://github.com/llvm/llvm-project/blob/main/clang/test/CodeGen/profile-filter.c#L10) not using a triple.